app/javascript/src/ui/invoices/InvoicesList.tsx

```diff
@@ -359,7 +359,7 @@ class InvoicesListComponent extends React.Component<Props, State> {
     const deleteModal = (renderProps: UpdateInvoiceRenderProps) => {
       const submit = async (form: Form) => {
         const success = await renderProps.submit(form);
-        success && this.setState({ selectedIds: [] });
+        success && this.setState({ selectedIds: [] }); // TODO: I don't think renderProps.submit ever returns true?
```

Comment:
It doesn't but should.  I've just been doing it quick and dirty by reading props after awaiting the write action.

Reply:
Do you want me to address that in this PR?

Reply:
Nah don't worry about it.